k-distribution/tests/regression-new/ceil-map-generated/ceil-map.k

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 K Team. All Rights Reserved.
+
+module CEIL-MAP-SYNTAX
+    imports MAP
+    imports INT
+
+    syntax Test ::= testRegular ( Map , Int , Int , Int , Int ) | testCell ( Int , Int , Int , Int , Int , Int ) | done ( K )
+
+    syntax Map ::= partialMap ( Map ) [function]
+    syntax Int ::= partialInt ( Int ) [function]
+
+endmodule
+
+module CEIL-MAP
+    imports CEIL-MAP-SYNTAX
+
+    configuration <k> $PGM:Test </k>
+                  <cellMap>
+                    <cellElem multiplicity="*" type="Map">
+                      <key>   0 </key>
+                      <cellA> 0 </cellA>
+                      <cellB> 0 </cellB>
+                    </cellElem>
+                  </cellMap>
+
+    rule <k> testRegular(MAP, K1, V1, K2, V2) => done( partialMap(MAP) (partialInt(K1) |-> partialInt(V1)) (partialInt(K2) |-> partialInt(V2)) ) </k>
```

Comment:
@ehildenb What's the best way to format long rules such as this?

Reply:
This is fine.